activestorage/app/models/active_storage/variant.rb

```diff
@@ -73,18 +73,21 @@ def key
     "variants/#{blob.key}/#{Digest::SHA256.hexdigest(variation.key)}"
   end
 
-  # Returns the URL of the variant on the service. This URL is intended to be short-lived for security and not used directly
-  # with users. Instead, the +service_url+ should only be exposed as a redirect from a stable, possibly authenticated URL.
-  # Hiding the +service_url+ behind a redirect also gives you the power to change services without updating all URLs. And
-  # it allows permanent URLs that redirect to the +service_url+ to be cached in the view.
+  # Returns the URL of the blob on the service. This returns a permanent URL for public files, and returns a
```

Comment:
```suggestion
  # Returns the URL of the blob variant on the service. This returns a permanent URL for public files, and returns a
```

Reply:
I updated `Variant` to reference the documentation for `Blob#url` to reduce duplication..